webpack.js

```diff
@@ -20,6 +20,8 @@ webpackConfig.entry = {
 	resultPage: { import: path.join(__dirname, 'src', 'resultPage.js'), filename: appId + '-resultPage.js' },
 }
 
+webpackConfig.output.publicPath = path.join('/', __dirname.split(path.sep).slice(-2).join(path.sep), 'js/')
```

Comment:
but why do that everywhere when you can do that in one place in the webpack config. It is the exact same thing (webpack public path).

Reply:
The reason is that we cannot predict the path at build time. Depending on how the  admin installs the app, we might be dealing with `apps-shared` or `apps` or even `apps-external`. We cannot know before we build the javascript bundle and ship it via the appstore. Thus we need to set the public path dynamically in the first file that gets loaded, which is all bundle entry files. Otherwise the subsequent chunks will not be found. :)

Reply:
ah sorry I didn't think of that. I was too deep in my local setup. That makes sense, thank you :)